sdk/communication/azure-communication-identity/azure/communication/identity/_shared/models.py

```diff
@@ -347,6 +369,88 @@ def __init__(self, bot_id, **kwargs):
         super().__init__(bot_id, **kwargs)
 
 
+class TeamsExtensionUserProperties(TypedDict):
+    """Dictionary of properties for a TeamsExtensionUserIdentifier."""
+
+    user_id: str
+    """The id of the Teams extension user."""
+    tenant_id: str
+    """The tenant id associated with the user."""
+    resource_id: str
+    """The resource id associated with the user."""
```

Comment:
```suggestion
    """The Communication Services resource id."""
```
maybe we can define it in this way, wdyt?

Reply:
done